src/main/kotlin/ar/edu/unahur/obj2/vendedores/Vendedor.kt

```diff
@@ -28,24 +28,60 @@ abstract class Vendedor {
   fun otrasCertificaciones() = certificaciones.count { !it.esDeProducto }
 
   fun puntajeCertificaciones() = certificaciones.sumBy { c -> c.puntaje }
+
+  abstract fun esInfluyente():Boolean
+  fun esGenerico() = certificaciones.any { v -> !v.esDeProducto }
 }
 
 // En los parámetros, es obligatorio poner el tipo
 class VendedorFijo(val ciudadOrigen: Ciudad) : Vendedor() {
   override fun puedeTrabajarEn(ciudad: Ciudad): Boolean {
     return ciudad == ciudadOrigen
   }
+
+  override fun esInfluyente(): Boolean {
+    return false
+  }
```

Comment:
Podés declarar a las consultas (métodos que devuelven) con `=` y así te ahorrás explicitar el tipo de dato.

```suggestion
  override fun esInfluyente() = false
```